internal/session/repository_session.go

```diff
@@ -682,6 +712,15 @@ func fetchConnections(ctx context.Context, r db.Reader, sessionId string, opt ..
 	return connections, nil
 }
 
+func fetchHostSetHost(ctx context.Context, r db.Reader, sessionId string, opt ...db.Option) (*SessionHostSetHost, error) {
+	const op = "session.fetchHostSetHost"
+	var hostSetHost *SessionHostSetHost
+	if err := r.SearchWhere(ctx, &hostSetHost, "session_id = ?", []interface{}{sessionId}, opt...); err != nil {
```

Comment:
Replace `interface{}` with `any`.
```suggestion
	if err := r.SearchWhere(ctx, &hostSetHost, "session_id = ?", []any{sessionId}, opt...); err != nil {
```

Reply:
Fixed [089532f](https://github.com/hashicorp/boundary/pull/2613/commits/089532f36b312f6679813a6dce84c2f57c108a9e)